### PR TITLE
Feat: Onboard dataset MIMIC-III

### PR DESCRIPTION
**Pull request by Naveen130**
## Description

This dataset includes 25 pipelines.

## Checklist

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved

Use the sections below based on what's applicable to your PR and delete the rest:

### Data Onboarding
- [x] I'm adding or editing a dataset
- [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
- [x] I put all my code inside  `datasets/<DATASET_NAME>` and nothing outside of that directory

### Code cleanup or refactoring
- [x] I'm refactoring or cleaning up some code

### CODE REVIEW

**Comment by Naveen130** [comment]
This pull request is to delete the existing dataset named mimicIII and add the dataset mimic_iii

**Comment by Naveen130** [comment]
Please review @adlersantos @nlarge-google

File: datasets/mimic_iii/pipelines/_images/run_script_kub/script.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    target_project_id: str,
+    target_bq_dataset: str,
+    timeout: int,
+):
+    client = bigquery_datatransfer_v1.DataTransferServiceClient()
+    transfer_config_name = f"{source_project_id}-{source_bq_dataset}-copy"
+    existing_config = find_existing_config(
+        client, target_project_id, transfer_config_name
+    )
+
```
Comment (nlarge-google) [inline datasets/mimic_iii/pipelines/_images/run_script_kub/script.py:45]: please remove blank lines in functions

File: datasets/mimicIII/pipelines/mimicIII/pipeline.yaml [hunk 2]
```diff
@@ -1,309 +0,0 @@
...
-        write_disposition: "WRITE_TRUNCATE"
-
-    - operator: "BigQueryToBigQueryOperator"
-      description: "Copy chartevents table from one bigquery project to another"
-      args:
-        task_id: "chartevents"
-        source_project_dataset_tables: "physionet-data.mimiciii_demo.chartevents"
-        destination_project_dataset_table: "bigquery-public-data-dev.mimicIII.chartevents"
-        write_disposition: "WRITE_TRUNCATE"
-
```
Comment (nlarge-google) [inline datasets/mimicIII/pipelines/mimicIII/pipeline.yaml:139]: Please remove blank lines between operators

**Review by nlarge-google — CHANGES_REQUESTED** [review]
Great job - just please remove blank lines and we should be good.  Thanks.

**Review by adlersantos — APPROVED** [review]

**Review by nlarge-google — APPROVED** [review]
Great Job Naveen!

**Pull request merged**